document_merge_service/api/unoconv.py

```diff
@@ -12,9 +15,10 @@
 # in testing 2 seconds is enough
 _min_timeout = 2
 
-# terminate_then_kill() takes 1 second in the worst case, so we have to use two seconds,
+# terminate_then_kill() takes 1 second in the worst case, so we have to use three seconds,
 # or the timeout won't be triggered before harakiri.
-_ahead_of_harakiri = 2
+# Increased to 3 seconds to be safe, we have reports of orphan soffice.bin in production.
+_ahead_of_harakiri = 3
```

Comment:
I heard from production that there are orphan soffice.bin, so I added another second.